activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Allow `cursor_column` in querying batches methods
+
+    Updated ActiveRecord::Batches#find_each, ActiveRecord::Batches#find_in_batches and
+    ActiveRecord::Batches#in_batches to receive `cursor_column`, the column to use for ordering.
+    By default the primary key is used.
```

Comment:
```suggestion
    By default the primary key is used.
    
    ```ruby
    Customer.find_each(cursor_column: :version) do |customer|
      NewsMailer.weekly(customer).deliver_now
    end
    ```
```

It's nice to see examples nested along with changelog items IMO